libs/ledger-live-common/src/families/hedera/tests/js-estimateMaxSpendable.test.ts

```diff
@@ -20,7 +21,7 @@ const account: Account = {
   blockHeight: 0,
   currency: {
     type: "CryptoCurrency",
-    id: "",
+    id: "" as CryptoCurrencyId,
```

Comment:
Shouldn't we put hedera if it's just an id we want, like this we do not need to cast ?